definitions/tools/pvacfuse.wdl

```diff
@@ -65,6 +72,10 @@ task pvacfuse {
     ~{if defined(fasta_size) then "-s ~{fasta_size}" else ""} \
     ~{if defined(downstream_sequence_length) then "-d ~{downstream_sequence_length}" else ""} \
     ~{if exclude_nas then "--exclude-NAs" else ""} \
+    ~{if defined(problematic_amino_acids) then "--problematic-amino-acids" else ""} ~{sep="," problematic_aa} \
```

Comment:
Technically if someone passed an empty array as the input this would trigger the param being added but with no value. We could check the `length` of `problematic_aa` similar to:
https://github.com/wustl-oncology/analysis-wdls/blob/4d3de66e39cb1fbfb10c5e9a5415f0c92df0cdcd/definitions/tools/generate_fda_tables.wdl#L46
or skip creating the intermediate variable and also do the `select` for `problematic_amino_acids` directly like:
https://github.com/wustl-oncology/analysis-wdls/blob/4d3de66e39cb1fbfb10c5e9a5415f0c92df0cdcd/definitions/tools/generate_fda_tables.wdl#L48

Reply:
Is this something that also needs to be fixed for `epitope_lengths_class_i` and `epitope_lengths_class_ii`?

Reply:
Ah yeah, those two are similar.  (Obviously it hasn't been an issue in practice since it's easy enough to avoid 🙂)